project.yaml

```diff
@@ -166,7 +166,7 @@ targets:
           - suse-sap-cloud:sles-15-sp2-sap
           - suse-sap-cloud:sles-15-sp5-sap
           - opensuse-cloud:opensuse-leap
-          - opensuse-cloud=opensuse-leap-15-4-v20231208-x86-64
+          - opensuse-cloud=opensuse-leap-15-6-v20240612-x86-64
```

Comment:
We should probably test on both 15-5 and 15-6, separately…

Reply:
Fair, added 15-5 to the exhaustive list too